submissions/Harnytskyi/memory_pair_game/script.js

```diff
@@ -0,0 +1,83 @@
+const CARDS = ["Kyiv", "Kharkiv", "Lviv", "Odesa", "Dnipro", "Poltava", "Zhytomyr", "Khmelnytskyi", "Luhansk", "Rivne", "Cherkasy", "Mykolaiv", "Simferopol", "Zaporizhzhia"];
+const GRID = document.getElementById("allCards");
+const FINISH_MATCH_CARDS = 8;
+const DELAY = 800;
+let matchCards = 0;
+let firstCard;
+
+function choseCards(cardArray) {
```

Comment:
```suggestion
function chooseCards(cards) {
```
1.typo 
2. i think naming like `cards` is enough
3. Change naming of function for more meaningful for example `chooseCardsRandomly`